TheEthicalHackerCup/Assets/Tutorial/Scripts/Tutorial.cs

```diff
@@ -5,57 +5,66 @@
 
 public class Tutorial : MonoBehaviour
```

Comment:
In this class, would it be possible to add methods that show/hide the tutorial?

I'm thinking those operations would be key for the eventual info button to re-show the tutorial and for conditionally hiding tutorials after they've been seen before